.bones/tasks/marky-8nzt.md

```
@@ -0,0 +1,94 @@
+
+Same cascade for each subsequent call. The existing marky-9m7o texts_transferred fix handles OOM in steps 6-9 (before toOwnedSlice), not the toOwnedSlice cascade itself.
+
+## Effort Estimate
+
+2-3 hours (scoped errdefer pattern + OOM regression test)
+
+## Implementation Checklist
+
+- [ ] After line 359 (headings toOwnedSlice success), add scoped errdefer:
+  \`\`\`zig
+  out_headings.* = stored_headings_list.toOwnedSlice(allocator) catch return error.OutOfMemory;
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)